src/Cache/CacheOperationTypes/CacheOperationTypes.php

```diff
@@ -1007,12 +1007,12 @@ public function __construct(_DictionaryGetResponse $responses = null, ?int $numR
         }
     }
 
-    public function values(): array
+    public function valuesArray(): array
```

Comment:
oh right, a dictionary in php is actually an array?

Reply:
Yeah, PHP's array is an array and a dictionary.